config/new_relic_otlp/cassandra/config.yaml

```diff
@@ -4,9 +4,8 @@ receivers:
     endpoint: localhost:7199
     target_system: cassandra,jvm
     collection_interval: 60s
-    properties:
-      # Attribute 'endpoint' will be used for generic_node's node_id field.
-      otel.resource.attributes: endpoint=localhost:7199
+    resource_attributes:
+      cassandra.endpoint: localhost:7199
```

Comment:
i don't know much about the new relic stuff. this doesn't break anything?

Reply:
It will if someone is searching on that key. Other than that it is fine. The keys can be arbitrary. This change follows the pattern we are using for other jmx based configs.

These configs should be used as a reference only, so I think the change is okay.